### ISSUE TITLE
Standard for object-storage

### ISSUE THREAD

**Issue by anjastrunk** [SovereignCloudStack/standards#725]
Object-storage is one mandatory service in SCS infrastructure, see #587. However, requirements for object-store are not yet defined. Similar to block storage, as an IaaS user, I want to know what capabilities a SCS object-store has. These capabilities include (on exhaustive list)

- [ ] ~API endpoint, as Object storage comes with two different APIs: S3 and Swift.~ S3 is mandatory, as defined in #587 
- [ ] Backup/Fail Safe capabilities
- [ ] Locality, region, availability zone

**Comment by mbuechse** [comment]
The idea was that #587 standardizes APIs, and in the case of object storage, we decided to make the S3 API mandatory and the Swift API recommended.

**Comment by anjastrunk** [comment]
> The idea was that #587 standardizes APIs, and in the case of object storage, we decided to make the S3 API mandatory and the Swift API recommended.

You are right and we have to standardize capabilities of components behind the API.

**Comment by toothstone** [comment]
I would like to add that [it came up during S3 discussions](https://github.com/SovereignCloudStack/minutes/blob/main/sig-standardization/20250918.md), that cloud users typically expect to have their Swift and S3 data to be in sync, i.e. that all their objects can be accessed from both services if both are provided. This has not been addressed by the SCS standards so far.
